packages/react-dom/src/server/ReactDOMNodeStreamRenderer.js

```diff
@@ -36,15 +37,21 @@ class ReactMarkupReadableStream extends Readable {
  * server.
  * See https://reactjs.org/docs/react-dom-server.html#rendertonodestream
  */
-export function renderToNodeStream(element) {
-  return new ReactMarkupReadableStream(element, false);
+export function renderToNodeStream(
+  element,
+  options: PartialRendererOptions | void,
```

Comment:
Should it be a public API? I suppose people won't know to pass this. Can it be autogenerated or something?

Reply:
We won't know when to generate the `prefix` because we don't want to prepend one unless necessary on the server, so regardless we would have to add an option.